libs/partners/huggingface/langchain_huggingface/llms/huggingface_pipeline.py

```diff
@@ -218,7 +230,6 @@ def from_model_id(
             model=model,
             tokenizer=tokenizer,
             device=device,
```

Comment:
```suggestion
            device=device,
            device_map=device_map,
```
would it make sense to keep this passthrough incase transformers uses it in future?